src/cli.ts

```diff
@@ -16,6 +16,41 @@ interface Repository {
     directory?: string
 }
 
+/**
+ * Flag indicating whether to include test tooling in the project, namely a test
+ * framework (mocha), extension API stubs, and a sample test.
+ *
+ * TODO: in the future, make this an option chosen by the user.
+ */
+const testsEnabled = true
```

Comment:
I think we should always include this, I don't see a benefit for a user to _not_ have this scaffolded. It would simplify the code below.

Reply:
I was thinking about users wanting to set up a different set of testing tools, so they might want to opt out.

Reply:
Possible, but I think for that "advanced" use case it's always easy to remove stuff from the scaffold after generating. E.g. if they wanted they could also use Webpack instead of Parcel, but we only generate a Parcel setup and not give options at generation time to keep it simple.